src/html-purify.js

```diff
@@ -9,7 +9,7 @@ See the accompanying LICENSE file for terms.
     var Parser = require('context-parser').Parser,
         tagAttList = require("./tag-attr-list"),
         derivedState = require('./derived-states.js'),
-        xssFilters = require('xss-filters'),
+        uriBlacklistFilter = require('xss-filters')._privFilters.yubl,
```

Comment:
i think it's okay to use this "private" filter. this is also used by secure-handlebars. you can consider them as "protected", and is stable for use for our packages.

@neraliu and I had a similar concern too. but finally settled to expose only the current set of public filters.
we sort of worried it's hard to explain to the public to avoid misuse, and most developers should be abstracted away from knowing/using this kind of "private/internal" filter independently